### ISSUE TITLE
Please remov calculations with 0 and 1

### ISSUE THREAD

**Issue by Mannshoch** [SecUSo/privacy-friendly-reckoning-skills#5]
calculations with 0 and 1 are not usefull.

**Comment by pandadeepimpact** [comment]
I agree with this. And their occurrence are quite significant, usually taking up 20-50% of the items given. I just tend to do more exercises to compensate, though it would be nice if this issue would be eliminated to save time.

**Comment by MikeSesketcharba** [comment]
Yes please
